### PR TITLE
#160609506 Setting up redux file structure

### PR DESCRIPTION
**Pull request by mendozabree**
#### What does this PR do

Setup redux 

#### Description of Task to be completed?

Currently, there is no state management for the application

This task adds redux to handle state management and creates folders for the actions, reducers and a store.

#### How should this be manually tested?

- The file structure can be seen [here](https://github.com/andela/ah-frontend-targaryen/tree/ch-setupredux-160609506/src)
#### What are the relevant pivotal tracker stories?

[#160609506](https://www.pivotaltracker.com/story/show/160609506)

#### Any background context you want to add?
N/A

#### Screenshots
image

### CODE REVIEW

**Review by RonKbS — APPROVED** [review]
LGTM

**Review by kakaemma — APPROVED** [review]
LGTM

File: src/reducers/rootReducer.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
+
+const rootReducer = combineReducers({
+// dummy object for reducer
+  f: () => ({}),
```
Comment (dondrzzy) [inline src/reducers/rootReducer.js]: @mendozabree Maybe you an actual name that will be used later. Such as `user` or `articles`. `f` may not mean much.

**Review by dondrzzy — COMMENTED** [review]
Please refactor

**Review by MuhanguziDavid — APPROVED** [review]
LGTM

File: src/reducers/rootReducer.js (hunk 1 above)
Comment (mendozabree) [reply src/reducers/rootReducer.js]: Thank you @dondrzzy, let me change that to `user`.

File: src/store.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import rootReducer from './reducers/rootReducer';
+
+const Store = createStore(
```
Comment (dondrzzy) [inline src/store.js]: Is there a reason for using a capital S?

Comment (mendozabree) [reply src/store.js]: No, there is no particular reason. Let me change that to `store`

**Pull request merged**